index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for randomatic 3.1
+// Project: https://github.com/jonschlinkert/randomatic
+// Definitions by: Sang Dang <https://github.com/sangdth>
+
+interface randomOptions {
+  chars: string;
```

Comment:
Shouldn't `chars` be `chars?`?

Reply:
In typescript, there is only `string`

Reply:
`chars?: string;` instead of `chars: string;` 😉

Reply:
![image](https://user-images.githubusercontent.com/326935/145130142-cbbac070-92d9-4f29-b5f3-25a61409e79a.png)

https://github.com/jonschlinkert/randomatic#options

Reply:
https://github.com/jonschlinkert/randomatic/blob/master/test.js#L107

Reply:
Ha ha I got it. How can I miss the second question mark :)) I will fix it.